bode/bode/resources/tasks/api.py

```diff
@@ -23,6 +24,13 @@ def post(self, task_data):
 
 @blueprint.route("/<task_id>")
 class TasksById(MethodView):
+    @blueprint.response(200, TaskSchema)
+    def get(self, task_id):
+        try:
+            return Task.query.get(task_id) or abort(404)
+        except DataError:
+            abort(404)
```

Comment:
1. if a correct UUID is passed -> returns a result if it exists. If not, responds with 404 (line 30)
2. if an incorrect UUID is passed -> `DataError` is caught and also responds with 404 (line 32)

@bkulawska please, test if your implementation of `DELETE` works. I got some errors in an `abort` function when I tried to pass `message` kwarg as you did.